src/screencast/view.ts

```diff
@@ -71,7 +71,7 @@ export class ScreencastView {
 
         for (const device of devicesArray) {
             // @ts-ignore ignoring as this is a static template.
-            templatedString += `<option deviceWidth=${device.screen.vertical.width} deviceHeight=${device.screen.vertical.height} ${ScreencastView.getDeviceCapabilities(device.capabilities)} value="${ScreencastView.getDeviceValueFromTitle(device.title)}">${device.title}</option>`;
+            templatedString += `<option deviceWidth=${device.screen.vertical.width} deviceHeight=${device.screen.vertical.height} ${ScreencastView.getDeviceCapabilities(device.capabilities)} userAgent=${escape(device['user-agent'])} value="${ScreencastView.getDeviceValueFromTitle(device.title)}">${device.title}</option>`;
```

Comment:
This is an odd design choice. I would expect us to use a `deviceId` as the select value and lookup the device in `screencast.ts`. Is there a reason we're using custom attributes?

Reply:
synced with Tony on this one, I don't see any blockers in changing this to used the deviceId. Another possible option as Tony mentioned would be to change the attributes to 'data-' attributes. e.g data-device-width. So I'm onboard with the one that you think will improve readability :-)

Reply:
> So I'm onboard with the one that you think will improve readability
My comment is less about readability and more about ease of extensibility. Currently the view holds the device data model while the viewmodel is the component that will make decisions based on device properties.

Reply:
Yes, I also favor putting an ID as the value so we can look up the information from the original source rather than having to copy it through the DOM (and consequently avoid updating this template each time we want to include more data).

Reply:
@hkal think you can incorporate that into this PR or do you want to break it out as a separate task?